### ISSUE TITLE
[Quest][NPC] Mimicking Nature's Call - spawning infinitive Frostgore

### ISSUE THREAD

**Issue by avarishd** [SallyWhitemane/Mograine#381]
**Description:** https://wotlkdb.com/?quest=11281 using the quest item provided spawns Frostgore (which is supposed to run to you but it does not, it just stays to the left of you while facing the ice cube). But you can use the horn multiple times to spawn as many Frostgore(s ) as you want.

**How to reproduce:** Pick up the quest, go to the location and spam the horn off cooldown. Can spawn indefinitely Frostgore(s)  https://wotlkdb.com/?npc=24173

**How it should work:** When you use the quest item Carved Horn https://wotlkdb.com/?item=33450 Frostgore should run to you from it's initial spawning position and you should be prevented from using this item if Frostgore is active/alive.

Quest: https://wotlkdb.com/?quest=11281
Item: https://wotlkdb.com/?item=33450
Frostgore[NPC]: https://wotlkdb.com/?item=33450

TL;DR
You can abuse the item to spawn milion Frostgore(s) and the event for it is not working (mob should run to you instead of staying to the side)

**Comment by SallyWhitemane** [comment]
Issue will be fixed with next update, thanks for contribution!
